motoman_driver/MotoPlus/debug.c

```diff
@@ -0,0 +1,70 @@
+//debug.c
+//
+
+#include "motoPlus.h"
+
+#define DEBUG_UDP_PORT_NUMBER	21789
+
+#define MAX_DEBUG_MESSAGE_SIZE	1024
+
+#define	SO_BROADCAST	0x0020
+
+#define MP_USER_LAN1	1	/* general LAN interface1 */
+#define MP_USER_LAN2	2	/* general LAN interface2(only YRC1000) */
+
+
+extern STATUS setsockopt(int s, int level, int optname, char* optval, int optlen);
+extern int mpNICData(USHORT if_no, ULONG* ip_addr, ULONG* subnet_mask, UCHAR* mac_addr, ULONG* default_gw);
```

Comment:
This is YRC1000 (and micro) only. So you probably want a conditional flag around this whole file.